Multisig/Cross-layer/Analytics/TrackingEvent.swift

```diff
@@ -83,6 +83,9 @@ enum TrackingEvent: String, Trackable {
     case generateOwnerOnboarding                    = "screen_owner_generate_info"
     case connectOwnerOnboarding                     = "screen_owner_walletconnect_info"
 
+    #warning("TODO: correct ledger onboarding tracking event name after alignment")
+    case ledgerOwnerOnboarding                      = "screen_owner_ledger_info"
```

Comment:
you can add it here https://github.com/gnosis/safe/wiki/Overview-screen-tracking
I used everywhere `ledger_nano_x`for tracking, so I would propose to change it here too for `screen_owner_ledger_nano_x_info `